README.md

```diff
@@ -1,41 +1,41 @@
+Deploy Action is intended to be used with [Deployment API keys](https://docs.astronomer.io/astro/api-keys). To use Deploy Action, you need to set the `ASTRONOMER_KEY_ID` and `ASTRONOMER_KEY_SECRET` environment variables in your GitHub Actions workflow. Astronomer recommends using GitHub Actions secrets to set the environment variables. An example workflow script is provided in **Deploy code example**. 
 
-#### Inputs
+## Configuration options
 
-You can configure the Deploy Actions behavior through a few options. None of the options are required.
+The following table lists the optional configuration options for Deploy Actions.
 
 | Name | Default | Description |
 | ---|---|--- |
-| `dag-deploy-enabled` | `false` | If true, only DAG files will be deployed when only DAG files are changed. __Only set this to true if DAG Deploys has been enabled for the Deployment you are deploying to__ |
-| `root-folder` | `.` | Specify the path to to Astro project folder that contains that 'dags' folder | 
-| `parse` | `false` | If true, DAGs will be parsed for errors before deployment |
-| `pytest` | `false` | If true, pytests will run before deployment |
-| `pytest-file` | (all tests run) | Specify custom pytest files to run with the pytest command. For example, you could specify `/tests/test-tags.py`
-| `force` | `false` | If true, your code will be force deployed, which skips tests and DAG parsing
-| `image-name` |  | Specify a custom locally built image to deploy |
+| `dag-deploy-enabled` | `false` | When set to `true`, DAG files are deployed only when the DAG files are changed. Only set this to `true` when DAG-only deploys are activated on the Deployment you are deploying to. |
+| `root-folder` | `.` | Specifies the path to to Astro project folder containing the 'dags' folder | 
+| `parse` | `false` | When set to `true`, DAGs are parsed for errors before deployment |
+| `pytest` | `false` | When set to `true`, pytests are run before deployment |
+| `pytest-file` | (all tests run) | Specifies the custom pytest files to run with the pytest command. For example, you could specify `/tests/test-tags.py`
+| `force` | `false` | When set to `true`, your code is force deployed without testing or parsing
+| `image-name` |  | Specifies a custom, locally built image to deploy |
 
 
-### Example Workflow File
+## Deploy code example
 
-The following example shows how the Deploy Action can be uesd to deploy your code to an Astro Deployment whenever code is pushed to the main branch of your projects repository. The ASTRONOMER_KEY_ID and ASTRONOMER_KEY_SECRET values come from your [Deployment API key](https://docs.astronomer.io/astro/api-keys).
+The following example shows how to use Deploy Action to deploy your code to an Astro Deployment whenever code is pushed to the main branch of your projects repository. The `ASTRONOMER_KEY_ID` and `ASTRONOMER_KEY_SECRET` environment variable values are defined by your [Deployment API key](https://docs.astronomer.io/astro/api-keys).
```

Comment:
```suggestion
```

Reply:
This is duplicative with the definition of the action itself, plus the information that we include earlier. I think it's pretty self explanatory.